Makefile

```diff
@@ -176,6 +179,24 @@ $(SUBMAN_TAR): subscription-manager
 $(SMBEXT_TAR): subscription-manager
 	tar czf $(SMBEXT_TAR) --transform 's,data/icons,src/subscription_manager/gui/data/icons,' subscription-manager/build_ext data/icons/hicolor
 
+# pybridge scenario: build and install the python bridge from cockpit repo
+ifeq ("$(TEST_SCENARIO)","pybridge")
+COCKPIT_PYBRIDGE_REF = main
+COCKPIT_WHEEL = cockpit-0-py3-none-any.whl
+
+$(COCKPIT_WHEEL):
+	# aka: pip wheel git+https://github.com/cockpit-project/cockpit.git@${COCKPIT_PYBRIDGE_REF}
+	rm -rf tmp/pybridge
+	git init tmp/pybridge
+	git -C tmp/pybridge remote add origin https://github.com/cockpit-project/cockpit
+	git -C tmp/pybridge fetch --depth=1 origin ${COCKPIT_PYBRIDGE_REF}
+	git -C tmp/pybridge reset --hard FETCH_HEAD
```

Comment:
hm am I missing anything, or wouldn't it be easier to simply do:
```
git clone --quiet --depth=1 https://github.com/cockpit-project/cockpit tmp/pybridge
```

Reply:
This was done to support a non-main COCKPIT_PYBRIDGE_REF, for testing against a proposed bridge change. But arbitrary refs  are probably not that important. I simplified it.